h/views/admin/search.py

```diff
@@ -2,6 +2,18 @@
 from pyramid.httpexceptions import HTTPFound
 from pyramid.view import view_config, view_defaults
 
+from h import models
+
+
+class NotFoundError(Exception):
+    pass
+
+
+@view_config(context=NotFoundError)
+def not_found(exc, request):
+    request.session.flash(str(exc), "error")
+    return HTTPFound(location=request.route_url("admin.search"))
```

Comment:
This is to handle showing a notification if an invalid username or groupid is submitted to the respective forms. Is there a better way of doing this? I copied and adapted it from the NIPSA forms.